resources/lang/ru_RU.yml

```diff
@@ -0,0 +1,271 @@
+# Translate by @udachin26
+# НЕ ТРОГАТЬ!
+file-version: 1
+# НЕ ТРОГАТЬ!
+
+
+BUTTON_JOIN_FACTION: "Вступить во фракцию"
+BUTTON_CREATE_FACTION: "Создать фракцию"
+BUTTON_TOP_FACTION: "Топ фракций"
+BUTTON_QUIT: "§4Выйти"
+BUTTON_BACK: "§4Назад"
+BUTTON_LEAVE_DELETE_FACTION: "§cВыйти/Удалить фракцию"
+BUTTON_VIEW_FACTION_MEMBERS: "Посмотреть участников"
+BUTTON_VIEW_FACTION_HOME: "Посмотреть точки домов"
+BUTTON_MANAGE_MEMBERS: "Управление участниками"
+BUTTON_MANAGE_FACTION: "Управление фракцией"
+BUTTON_SEND_INVITATION: "Отправить приглашение"
+BUTTON_INVITATION_PENDING: "Приглашение отправлено"
+BUTTON_REQUEST_PENDING: "Запрос получен"
+BUTTON_LISTING_HOME: "{{name}}\nx:{{x}} | y:{{y}} | z:{{z}}"
+BUTTON_MODAL_YES: "§2Да"
+BUTTON_MODAL_NO: "§4Нет"
+BUTTON_CHANGE_RANK: "Сменить ранг"
+BUTTON_KICK_OUT: "§4Кикнуть из фракции"
+BUTTON_TRANSFER_PROPERTY: "§cПередача права собственности"
+BUTTON_DELETE_INVITATION: "§cУдалить приглашение"
+BUTTON_ACCEPT_REQUEST: "Принять запрос"
+BUTTON_REFUSE_REQUEST: "Отклонить запрос"
+BUTTON_MANAGE_ALLIANCE: "Управление союзами"
+BUTTON_CHANGE_DESCRIPTION: "Изменить описание"
+BUTTON_CHANGE_MESSAGE: "Изменить сообщение фракции"
+BUTTON_CHANGE_VISIBILITY: "Изменение видимости"
+BUTTON_CHANGE_PERMISSION: "Изменить разрешения"
+BUTTON_BREAK_ALLIANCE: "§4Разорвать союз"
+BUTTON_CHANGE_LANGUAGE: "Изменить язык"
+BUTTON_LEVEL_UP: "Повысить уровень"
+BUTTON_LEVEL_UP_READY: "§l§eГотовы к повышению уровня!\n§r§lУровень {{level}}"
+BUTTON_LEVEL_UP_ADVANCE: "{{advance}}\n§r§lУровень {{level}}"
+BUTTON_LEVEL_UP_MAX: "§cМаксимальный уровень"
+BUTTON_LEADERBOARD: "Таблица лидеров"
+BUTTON_LEADERBOARD_FACTIONLEVEL: "Сортировать по уровню"
+BUTTON_LEADERBOARD_FACTIONPOWER: "Сортировать по силе"
+
+LEVEL_UP_TITLE_MAIN: "Повысить уровень"
+LEVEL_UP_CONTENT_MAIN_MAX: "§r§4Вы достигли предела уровня"
+LEVEL_UP_CONTENT_MAIN: "§r§nНаграда следующего уровня : §r\n §5>> §9{{name}} x{{value}}"
+CHANGE_LANGUAGE_TITLE: "Изменить язык"
+MANAGE_ALLIANCE_MAIN_TITLE: "Управление союзом"
+MANAGE_ALLIANCE_TITLE: "Управление союзом - {{name}}"
+CHANGE_DESCRIPTION_TITLE: "Управление описанием"
+CHANGE_DESCRIPTION_INPUT_CONTENT: "Введите описание здесь"
+CHANGE_MESSAGE_TITLE: "Управление сообщением фракции"
+CHANGE_MESSAGE_INPUT_CONTENT: "Введите сообщение здесь"
+CHANGE_PERMISSION_TITLE: "Управление разрешениями"
+CHANGE_VISIBILITY_TITLE: "Изменить видимость"
+CHANGE_VISIBILITY_STEP: "Выбрать видимость"
+CHANGE_VISIBILITY_EXPLICATION: "§n§7Public :§r\n  The faction will be free to all and everyone can come and go as they please\n \n§n§7Private :§r\n  The faction will be closed to invitations and players will have to be invited by a member\n \n§n§7By invitation :§r\n  Players will be able to send invitations to the faction and the faction will be able to invite players"
```

Comment:
Any reason why this line is not translated?

Reply:
human factor